src/constructs/bindings.md

```diff
@@ -2,16 +2,19 @@
 title: Bindings
 ---
 
-## On the Subject of `let` and `let rec`
+## On the Subject of `let`, `let rec`, and `let mut`
```

Comment:
I think this order makes a little more sense when looking at this for the first time: immutable stuff -> mutable stuff -> recursive stuff.
```suggestion
## On the Subject of `let`, `let mut`, and `let rec`
```

Reply:
Also this title is still lame. I gave these all a default of `Of the Subject of...` a few years ago thinking that since it was a bit of a silly title, I'd actually go back and change them all. Of course, I never did. Maybe just doing this would be better?

```suggestion
## `let`, `let mut`, and `let rec`
```